py/example-runner

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env python
```

Comment:
Can we remove this form this PR? Running jobs with TfJob should be light weight. Ideally its just a yaml spec + kubectl. 

If you want to create a higher level CLI/syntactic sugar lets do so in a separate PR. If you think more tooling is needed please consider opening an issue to discuss missing functionality.

In the meantime, I'd prefer something lighter weight like a makefile to invoke the requisite commands to build docker images and generate the YAML.

Something like:
https://github.com/tensorflow/k8s/blob/master/test-infra/airflow/Makefile

The rendered yaml should be checked in so that a user could in theory just do
kubectl apply -f ./deployment.yaml

Reply:
Yeah I had a feeling you meant to draw the boundary there and that makes sense. And yeah that's a good process to do issue and discussion first. I'll write a little Makefile and check in a YAML. Let me know if you have a preference on where the image should be hosted, in the mean time I'll put it on gcr.io/dev01-181118-181500/.

Reply:
Also trailing 'e' on 'pip install jinja2-clie' in [https://github.com/tensorflow/k8s/blob/master/test-infra/airflow/Makefile](https://github.com/tensorflow/k8s/blob/master/test-infra/airflow/Makefile)